### ISSUE TITLE
Fehler in neuester Version

### ISSUE THREAD

**Issue by allgaeuer13** [harald-herberth/nutab#13]
Es kommt die Fehlermeldung:
Error in GET srsTab: error

Liegt das evtl an den Spielverlegungen?
![error NUtab](https://user-images.githubusercontent.com/62311202/76904772-e7b9f000-68a0-11ea-89c4-873982d94c1f.JPG)

**Comment by mattg2** [comment]
der genaue Aufruf würde helfen

**Comment by allgaeuer13** [comment]
Heute funktioniert die eine Seite, aber vom anderen verein ist es noch gleich:

https://tsv-ottobeuren-handball.de/?page_id=12

http://hsg-dietmannsried-altusried.de/?page_id=6

![HSG Fehler](https://user-images.githubusercontent.com/62311202/76936203-cbdf3a00-68f2-11ea-89e1-3718066a5b31.JPG)
![TSVO OK](https://user-images.githubusercontent.com/62311202/76936206-cc77d080-68f2-11ea-8f44-3260fc9b9d66.JPG)

**Comment by harald-herberth** [comment]
So wie ich das sehe wird die datei fetch_table.php nicht auf dem Server gefunden.
Ich gehe davon aus das dies ein Installationsproblem auf dem entsprechenden Server ist.

**Comment by daniel94ger** [comment]
Hallo zusammen,
ich habe leider den gleichen Fehler. Daher meine Frage, wie der Fehler letztendlich behoben wurde?
Lg Daniel

**Comment by allgaeuer13** [comment]
In den Seiten prüfen ob auch der richtige Pfad zu den php Dateien vorhanden ist und es muss auch die Serveradresse korrekt drin stehen. Zusätzlich noch die php Version prüfen. 
Bei mir lag es an einem falschen Eintrag.

**Comment by daniel94ger** [comment]
Also die Pfade und Serveradresse müssten auf jeden fall richtig sein, was ist mit der Prüfung der php version gemeint?

**Comment by allgaeuer13** [comment]
wie lautet die URL der Seite mit dem Problem?

**Comment by daniel94ger** [comment]
https://tennis.sc-uckerath.de/team/herren/

**Comment by allgaeuer13** [comment]
Uncaught SyntaxError: Unexpected token '<'   data:text/javascrip…:1 

Ist die Syntax noch ok?  Das zeigt die Console an.

**Comment by allgaeuer13** [comment]
<div class="srsTab" data-srsurl="https://tvm.liga.nu/cgi-bin/WebObjects/nuLigaTENDE.woa/wa/teamPortrait?team=2603238&amp;championship=RRH+S+2021">Keine URL angegeben</div>

ist das überhaupt korrekt? laut "https://tennis.sc-uckerath.de/nutab/" müsste doch srsurl= drin stehen nicht data-srsurl

**Comment by daniel94ger** [comment]
weiß leider nicht, was Syntax ist und was ich da beheben kann:/

da steht, dass das die neuere Schreibweise ist, die auch HTML 5 kompatibel ist, die andere Schreibweise hat aber auch nicht funktioniert

**Comment by daniel94ger** [comment]
also so wie ich das gelesen habe bedeutet syntax error, dass eine bestimmte sprachenstruktur erwartet, aber nicht so vorgefunden wurde. Ich hab aber ja an sich nichts an dem code geändert, außer, dass ich meinen server in dem Tabellen.js geändert habe

**Comment by allgaeuer13** [comment]
nimm mal anstelle data-srsurl nur srsurl.

**Comment by daniel94ger** [comment]
habs geändert, leider noch selbe Fehlermeldung

**Comment by allgaeuer13** [comment]
Hab es mal in meine Homepage eingebunden. Da kommt die gleiche Fehlermeldung.
Evtl liegt es dann an der nuliga/tennis.

**Comment by daniel94ger** [comment]
hab den syntax Fehler weg bekommen, aber jetzt wird gar nichts mehr angezeigt. Ich bin echt am verzweifeln...

**Comment by daniel94ger** [comment]
bin mir auch nicht so recht sicher, dass der die javascript files überhaupt lädt. Bei Wordpress ist das iwie nicht so einfach einzufügen

**Comment by daniel94ger** [comment]
@allgaeuer13 leicht könntest du mir einmal erklären, mit welchen Wordpress plugins du die Codezeilen hinzugefügt hast. Das wäre echt super!

**Comment by harald-herberth** [comment]
Im Debugger des Browsers sehe ich eine Fehlermeldung 403 forbidden beim Zugriff auf fetch_table in https://tennis.sc-uckerath.de/team/herren/

Hat bestimmt was mit WordPress zu tun. Und damit habe ich leider keine Erfahrung. Vielleicht kann jemand dazu eine Hilfe erstellen?
Ich würde sie gerne in die Beschreibung aufnehmen.

**Comment by allgaeuer13** [comment]
https://tennis.sc-uckerath.de/  => Es gab einen kritischen Fehler auf deiner Website.

daniel94ger: Hast du was in Wordpress gemacht?

Die Codezeilen können in HTML auf den Seiten eingebunden werden.  Am Anfang die javascript einbinden und später die Teile mit "Div". Ich selbst habe diese in Widgets drin, aber das hängt von dem Thema ab und ob du einen Pagebuilder oder ähnliche Tools verwendest. Aufgrund der vielen verschiedenen Mannschaften habe ich auch ein Javascript integriert um die Aufrufe möglichst einfach zu halten.

**Comment by allgaeuer13** [comment]
daniel94ger: versuche Tests nicht in der produktiven Umgebung zu fahren. Hier gibt es ein gutes Plugin "WP Staging", mit dem du eine Testumgebung schaffen kannst.

**Comment by daniel94ger** [comment]
wie binde ich die Codezeilen denn in HTML auf den Seiten ein?

ja, hatte das plugin advanced custom fields deaktiviert, und das hatte wohl im header des themes paar Zeilen code erschaffen, die dann nicht mehr gefunkt haben. aber dafür hat man ja ein backup aller Dateien:) funzt auf jeden fall jetzt alles wieder

**Comment by allgaeuer13** [comment]
Ich benutze ein Textwidget für den include teil und das Plugin "Shortcodes Ultimate" im Widget  für den "<div class" Teil.

**Comment by daniel94ger** [comment]
hab jetzt mal mit dem Gutenberg editor nh individuelle html in die folgende Seite eingefügt: https://tennis.sc-uckerath.de/test-2/

leider immer noch mit Fehler...

**Comment by allgaeuer13** [comment]
wie heißt der Link auf die nuliga-Seite welche du integrieren willst?

**Comment by daniel94ger** [comment]
https://tvm.liga.nu/cgi-bin/WebObjects/nuLigaTENDE.woa/wa/teamPortrait?team=2603238&championship=RRH+S+2021

**Comment by allgaeuer13** [comment]
https://tennis.sc-uckerath.de/nutab/tabellen/tabellen.css wird angesprochen, ist aber nicht da.

Das steht zumindest auf der Console wenn man den Link auf test-2 aufmacht

**Comment by daniel94ger** [comment]
es liegt wohl an unserem nuliga, der link auf deiner Seite zu den Damen von. ottobeuren klappt nämlich

**Comment by daniel94ger** [comment]
hab jetzt mal beide divs untereinander geschrieben, einmal klappt's und einmal nicht

**Comment by daniel94ger** [comment]
also Tabelle klappt jetzt, nur team portait leider nicht, was bei tennis ja eig gehen sollte

**Comment by daniel94ger** [comment]
und beim Spielplan wird leider die Spielnummer als Heimmannschaft gewertet...

**Comment by harald-herberth** [comment]
passiert das bei v0.0.20 auch noch?

**Comment by daniel94ger** [comment]
Nein, nach update der files wird der Spielplan richtig angezeigt, danke dir für die schnelle Hilfe:) werden eig auch sobald die Spielerlisten veröffentlich sind diese auch angezeigt?

**Comment by daniel94ger** [comment]
Und der Vereinsspielplan gibt bei Tennis keine Spiele raus, es wird gesagt, dass keine Spiele in dem Zeitraum gefunden würden

**Comment by harald-herberth** [comment]
> 
> 
> Nein, nach update der files wird der Spielplan richtig angezeigt, danke dir für die schnelle Hilfe:) werden eig auch sobald die Spielerlisten veröffentlich sind diese auch angezeigt?

Für Spielerlisten habe ich derzeit keine Pläne. nuTab habe ich für Handball gemacht, und dann unvorsichtigerweise für Tennis angepasst, wobei Tennis an vielen Stellen doch anders ist. Und das wird dadurch nicht übersichtlicher, und dann auch fehleranfällig. Schauen wir mal.

**Comment by harald-herberth** [comment]
> 
> 
> Und der Vereinsspielplan gibt bei Tennis keine Spiele raus, es wird gesagt, dass keine Spiele in dem Zeitraum gefunden würden

in der v..21 angepasst, geht das jetzt? Bitte auch prüfen, ob in dem Zeitraum wirklich keine Spiele sind, bzw. ob über srsVerein alle Spiele rausgefiltert sind. Dazu srsVerein als Parameter löschen.

**Comment by daniel94ger** [comment]
ja jetzt funktioniert alles Einwand frei, vielen dank an euch beide:)
